openmetadata-ui/src/main/resources/ui/src/pages/BotDetailsPage/BotDetailsPage.tsx

```diff
@@ -62,34 +65,32 @@ const BotDetailsPage = () => {
     }
   };
 
-  const fetchBotsData = () => {
-    setIsLoading(true);
-    getUserByName(botsName)
-      .then((res) => {
-        if (res) {
-          setBotsData(res);
-        } else {
-          throw jsonData['api-error-messages']['unexpected-server-response'];
-        }
-      })
-      .catch((err: AxiosError) => {
-        showErrorToast(
-          err,
-          jsonData['api-error-messages']['fetch-user-details-error']
-        );
-        setIsError(true);
-      })
-      .finally(() => setIsLoading(false));
+  const fetchBotsData = async () => {
+    try {
+      setIsLoading(true);
+      const botResponse = await getBotByName(botsName);
+
+      const botUserResponse = await getUserByName(
```

Comment:
Isn't this what we have in `botResponse`?

Reply:
No, botResponse will have the bot entity-related data and `botUser` entity reference. so we will have to take the `botUser` fqn and get the user entity data.